### PR TITLE
add function helm-swoop--goto-line-action and helm-swoop--goto-line-a…

### PR DESCRIPTION
**Pull request by jixiuf**
add function helm-swoop--goto-line-action and helm-swoop--goto-line-action-hook

demo for evil user:
(defun push-mark-after-helm-swoop()
  (isearch-update-ring helm-pattern evil-regexp-search))
(add-hook 'helm-swoop--goto-line-action-hook 'push-mark-after-helm-swoop)
#101

### CODE REVIEW

File: helm-swoop.el [hunk 1]
```diff
@@ -150,6 +150,9 @@
     (switch-to-buffer $buf))
   "Change the way to split window only when `helm-swoop' is calling")
 
+(defvar helm-swoop--goto-line-action-hook nil
```
Comment (syohex) [inline helm-swoop.el]: `--` which means private variable/function should not be used if it is used by users, not developers. And then `defcustom` is better than `defvar`

**Comment by jixiuf** [comment]
updated

File: helm-swoop.el [hunk 2]
```diff
@@ -150,6 +150,11 @@
     (switch-to-buffer $buf))
   "Change the way to split window only when `helm-swoop' is calling")
 
+(defcustom helm-swoop-goto-line-action-hook nil
+  "hooks run after `helm-swoop--goto-line"
+  :group 'helm-swoop
+  :type '(repeat string))
```
Comment (syohex) [inline helm-swoop.el]: `:type` should be `'hook`, not `'string`.

**Comment by jixiuf** [comment]
updated

**Pull request closed without merging**